cache/cache.go

```
@@ -83,7 +83,7 @@ func (c *BigCache) GetLiquidityRules(nodePubkey string) (map[uint64][]nodeguard.
 
 	entry, err := c.cache.Get(nodePubkey)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

shouldn't at least log it so we don't swallow the error?

---

I'll add it